src/common/popup/utils/format.js

```diff
@@ -14,8 +14,13 @@ function branch(ticket) {
   return `${ticket.type || 'feature'}/${ticket.id}-${normalize(ticket.title)}`;
 }
 
+function command(ticket) {
+  return `git checkout -b ${branch(ticket)}  && git commit --allow-empty -m"${commit(ticket)}"`;
```

Comment:
i gave a bad example in my proposal-issue ;) (for some reason i never write that space, to save a byte)